apps/ejabberd/src/ejabberd.app.src

```diff
@@ -39,7 +39,8 @@
                   fast_tls,
                   usec,
                   uuid,
-                  xmerl
+                  xmerl,
+                  worker_pool
```

Comment:
`worker_pool` provide more worker checkout types and the one used by this module is not even available in `poolboy`. Also poolboy-like checkout is much faster in `worker_pool`. Another thing is that `worker_pool` is already used in our ODBC driver, I don't really know why this change shows up in diff. Maybe I should rebase this branch...